src/test/groovy/net/fortuna/ical4j/model/DateTimeSpec.groovy

```diff
@@ -112,10 +112,10 @@ class DateTimeSpec extends Specification {
                 }
         ]
 
-        dateTimeString << ['20110327T000000', '20110327T000000']
+        dateTimeString << ['20110327T000000Z', '20110327T000000Z']
```

Comment:
Is this change required for the gradle upgrade? By using UTC format DATE-TIME i suspect it won't properly test the custom timezone..

Reply:
not sure if it was change to Groovy version or a plugin but for some reason this test was failing after the changes and specifying UTC fixed the problem